.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
```

Comment:
I write with reference to https://sarunw.com/posts/github-actions-for-ios-projects/